pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -867,6 +868,17 @@ class MultiControlledX(Operation):
     * Gradient recipe: None
 
     Args:
+        New args:
```

Comment:
The docstring needs to fit a specific syntax so Sphinx knows how to display the argument list.  You can check out the rendered webpage for this docstring at https://pennylane--2121.org.readthedocs.build/en/2121/code/api/pennylane.MultiControlledX.html . 

Instead of having a `New args` and `Old args` list, we can instead update the description of `control_wires` to indicate the keyword is being deprecated. Maybe something like:

```
        control_wires (Iterable[Any]): Deprecated way to indicate the control wires. Now users should use
             ``wires`` to indicate both the control wires and the target wire.
```